core/templates/dev/head/css/oppia.css

```diff
@@ -197,6 +207,7 @@ p {
   width: 80%;
 }
 
+
```

Comment:
You don't need to add this extra line.

(Note that, before you commit, you can use "git diff" to see what lines you changed.)